yarn.lock

```diff
@@ -13358,22 +13398,10 @@ write-file-atomic@^4.0.1, write-file-atomic@^4.0.2:
     imurmurhash "^0.1.4"
     signal-exit "^3.0.7"
 
-ws@8.13.0:
-  version "8.13.0"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-8.13.0.tgz#9a9fb92f93cf41512a0735c8f4dd09b8a1211cd0"
-  integrity sha512-x9vcZYTrFPC7aSIbj7sRCYo7L/Xb8Iy+pW0ng0wt2vCJv7M9HOMy0UoN3rr+IFC7hb7vXoqS+P9ktyLLLhO+LA==
-
-ws@^6.1.0:
-  version "6.2.2"
```

Comment:
This is coming from storybook (via puppeteer-core), so I am not concerned about it. Storybook 8 removes the dependency on puppeteer-core. Opened https://github.com/open-path/Green-River/issues/6261 since lots of vuln issues stem from storybook